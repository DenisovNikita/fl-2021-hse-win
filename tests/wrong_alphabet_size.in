Alphabet -2
a
b
States 2
Edges 4
0 to 1 by a
0 to 1 by b
1 to 0 by a
1 to 0 by b
Terminals 1
0
