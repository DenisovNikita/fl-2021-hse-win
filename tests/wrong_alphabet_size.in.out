Graph is not DFA
Incredible bad input:
alphabet_size < 0
